src/lib/utils/theme/typography/Typography.ts

```diff
@@ -0,0 +1,128 @@
+import { TypographyOptions } from '@mui/material/styles/createTypography'
+import { Palette } from '@mui/material/styles/createPalette'
+
+const defaultStyles = {
+  fontStyle: 'normal',
+  letterSpacing: '0px',
+}
+
+export const typographyOptions: (
+  palette: Palette,
+) => TypographyOptions = palette => ({
+  fontFamily: ['Lato', 'sans-serif'].join(','),
+  headline1: {
+    ...defaultStyles,
+    fontSize: '24px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    marginTop: '28px',
+  },
+  headline2: {
+    ...defaultStyles,
+    fontSize: '20px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    marginTop: '24px',
+  },
+  headline3: {
+    ...defaultStyles,
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '20px',
+    marginTop: '20px',
+  },
+  body1: {
+    ...defaultStyles,
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    marginTop: '24px',
+  },
+  body1Italic: {
+    ...defaultStyles,
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    marginTop: '24px',
+    fontStyle: 'italic',
+  },
+  body2: {
+    ...defaultStyles,
+    fontSize: '12px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    marginTop: '24px',
+  },
```

Comment:
`body2` is used internally by MUI for some components like Pagination. We really want it to just be a smaller version of body1.